3.interpret_models/log_reg_plate_3_3p_cp_renorm_fs_data/feature_importances/nbconverted/seperate_compartments.py

```diff
@@ -0,0 +1,141 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# # Here the features are seperated according to compartments for later analysis
+
+# In[1]:
+
+
+# %%--%%| <qGnYViiwRD|SyZ3qa8iz3>
+r"""°°°
+## Imports
+°°°"""
+
+
+# In[2]:
+
+
+import pandas as pd
+from pathlib import Path
+import itertools
+
+
+# ## Find the git root Directory
+
+# In[3]:
+
+
+# Get the current working directory
+cwd = Path.cwd()
+
+if (cwd / ".git").is_dir():
+    root_dir = cwd
+
+else:
+    root_dir = None
+    for parent in cwd.parents:
+        if (parent / ".git").is_dir():
+            root_dir = parent
+            break
+
+# Check if a Git root directory was found
+if root_dir is None:
+    raise FileNotFoundError("No Git root directory found.")
```

Comment:
same comment about `__file__` metadata and creating a function

Reply:
I left the followup comment above